doc/api/process.md

```diff
@@ -1429,15 +1428,20 @@ Android)
 The `process.stderr` property returns a [Writable][] stream equivalent to or
 associated with `stderr` (fd `2`).
 
-`process.stderr` and `process.stdout` are unlike other streams in Node.js in
-that they cannot be closed (calling [`end()`][] will throw an Error), they never
-emit the [`'finish'`][] event, and writes can block when output is redirected to
-a file (although disks are fast and operating systems normally employ write-back
-caching so it should be a very rare occurrence indeed.)
+Note: `process.stderr` and `process.stdout` are differ from other streams in
+Node.js in a couple ways:
+1. They cannot be closed ([`end()`][] will throw).
+2. They never emit the [`'finish'`][] event.
+3. Writes _can_ block when output is redirected to a file.
+  - (Although disks are fast and operating systems normally employ write-back
+    caching so this is very uncommon.)
+4. Writes on UNIX __will__ block by default if output is going to a TTY
+   (a terminal).
+5. Windows functionality differs. Writes block except when output is going to a
```

Comment:
ok maybe cc @saghul on this, since I don't really know where to look in libuv besides `uv_stream_set_blocking`. It appears to set `UV_HANDLE_BLOCKING_WRITES` on Windows but I don't know how to for sure tell if it only affects pipes.

A search only shows results in `win/pipe.c` and `win/stream.c`.